### DISCUSSION TITLE
serial port

### DISCUSSION THREAD

**Discussion by whbg2022** [HefnySco/OBAL#13] [Q&A] [answered]
Hi HefnySco,

very nice work, thank you!

a quick question: in your design it is only possible to have one serial port physical connection? right now your design is for GPS.

I am just starting to read about ArduoPilot and this is something I wonder about in your design.

thx.

**Comment by HefnySco** [comment, marked as answer]
RPI-4 have a serial port in pins 14,15 and any usb can be used with Serial to USB.
![image](https://user-images.githubusercontent.com/4184373/163655083-2a81a902-3658-43d4-918f-33b1f2c1a1a1.png)

For RPI-Zero there is a serial port 0 in pins 14,15 and the USB can be used to connect OTG then another Serial to USB.

**Reply by whbg2022** [reply]
I see. thank you for the explanation.
